ee/hogai/tool.py

```diff
@@ -124,6 +124,7 @@ class MaxTool(AssistantContextMixin, BaseTool):
     It will be formatted like an f-string, with the tool context as the variables.
     For example, "The current filters the user is seeing are: {current_filters}."
     """
+    show_tool_call_messages: bool = Field(description="Whether to show tool call messages.", default=False)
```

Comment:
Nit: since we output a single tool call message, the singular "message" would make more sense.

```suggestion
    show_tool_call_message: bool = Field(description="Whether to show a tool call message in the UI.", default=False)
```